changelog.md

```diff
@@ -480,6 +490,13 @@
 ### <a name="1.0.0"></a> 1.0.0
 - GA SDK.
 
+## Known regressions
+
+Below is a list of any know regressions or issues affecting the [recommended minimum version](#recommended-version):
+
+* When executing queries and using Direct mode but the SDK needs to obtain the Query Plan from Gateway (more details on the [performance guide](performance-tips.md#hosting-recommendations)), the query execution will also go through Gateway instead of honoring Direct mode.
```

Comment:
Ideally this section needs to be more actionable. 

As CX the follow-up will always be what's the impact when will these get addressed, related GitHub issue etc... 
We need to capture these aspects very expliclty.

Reply:
Also as CX are these issue exists before min-recommended version?

Reply:
Refactored as a table with potential mitigation and tracking link.

This issues might be from before the recommended version, these are issues we know are present there because they were reported/detected on it.